swri_transform_util/package.xml

```diff
@@ -13,13 +13,12 @@
   <url>https://github.com/swri-robotics/marti_common</url>
   
   <buildtool_depend>catkin</buildtool_depend>
-  
+
   <depend>boost</depend>
   <depend>diagnostic_msgs</depend>
   <depend>geographic_msgs</depend>
   <depend>geometry_msgs</depend>
   <depend>libgeos++-dev</depend>
-  <depend>libopencv-dev</depend>
```

Comment:
If we're not using the system-provided OpenCV 2 anymore, it makes sense to remove this, but would it make sense to add a rosdep key for the ROS-provided OpenCV 3?